content/en/docs/tasks/configure-pod-container/assign-pod-level-resources.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Assign Pod-level CPU and memory resources
+content_type: task
+weight: 30
+min-kubernetes-server-version: 1.32
+---
+
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="PodLevelResources" >}}
+
+This page shows how to specify CPU and memory resources for a Pod at pod-level in
+addition to container-level resource specifications. A Kubernetes node allocates
+resources to a pod based on the pod's resource requests. These requests can be
+defined at the pod level or individually for containers within the pod. When
+both are present, the pod-level requests take precedence.
+
+Similarly, a pod's resource usage is restricted by limits, which can also be set at
+the pod-level or individually for containers within the pod. Again,
+pod-level limits are prioritized when both are present. This allows for flexible
+resource management, enabling you to control resource allocation at both the pod and
+container levels.
+
+In order to specify the resources at pod-level, it is required to enable
+`PodLevelResources` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+For Pod Level Resources:
+* Priority: When both pod-level and container-level resources are specified,
+  pod-level resources take precedence.
+* QoS: Pod-level resources take precedence in influencing the QoS class of the pod.
+* OOM Score: The OOM score adjustment calculation considers both pod-level and
+  container-level resources.
+* Compatibility: Pod-level resources are designed to be compatible with existing
+  features.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+The `PodLevelResources` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled
+for your control plane and for all nodes in your cluster.
+
+## Create a pod with memory requests and limits at pod-level
+
+To specify memory requests for a Pod at pod-level, include the `resources:requests:memory`
+field in the Pod spec manifest. To specify a memory limit, include `resources:limits:memory`.
+
+In this exercise, you create a Pod that has one Container. The Pod has a
+memory request of 100 MiB and a memory limit of 200 MiB. Here's the configuration
+file for the Pod:
+
+{{% code_sample file="pods/resource/pod-level-memory-request-limit.yaml" %}}
+
+The `args` section in the configuration file provides arguments for the Container when it starts.
```

Comment:
Even though this adapts existing content, let's use the official name for this format:
```suggestion
The `args` section in the manifest provides arguments for the container when it starts.
```